server/datastore/mysql/migrations/tables/20220708095046_AddUniqconstraintSoftwareIDOnSoftwareCVE.go

```diff
@@ -0,0 +1,29 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220708095046, Down_20220708095046)
+}
+
+func Up_20220708095046(tx *sql.Tx) error {
+	fmt.Println("Adding unique constraint on (cve, software_id) to software_cve table...")
+	_, err := tx.Exec(`
+	ALTER TABLE software_cve ADD CONSTRAINT unq_software_id_cve UNIQUE (software_id, cve), ALGORITHM=INPLACE, LOCK=NONE;
```

Comment:
Not sure how out there this is, but before adding a constraint, should we make sure there are no duplicates?

Reply:
Would the whole row be duplicated in that case?